backend/baseclass.pm

```diff
@@ -567,7 +567,7 @@ sub reset_consoles {
 
     # we iterate through all consoles
     for my $console (keys %{$testapi::distri->{consoles}}) {
-        #next if ($console eq 'x3270');
+        next if $self->console($console)->{args}->{persistent};
```

Comment:
IMHO this is the important part that should be emphasized in your git commit message subject line as well as the PR subject. You are not just adding the option, you are adding support for persistent consoles. How about instead of "Add persistent option to console specific parameters" you write "Allow consoles to persist over reset"?